docs/intro.rst

```diff
@@ -0,0 +1,217 @@
+      \text{of multiple channels}}} \underbrace{\color{red}{\prod_{\singleconstr \in \constrset} c_{\singleconstr}(a_{\singleconstr} |\, \singleconstr)}}_{\substack{\text{constraint terms}\\%
+      \text{for }\unicode{x201C}\text{auxiliary measurements}\unicode{x201D}}},
+
+where within a certain integrated luminosity we observe :math:`n_{cb}`
+events given the expected rate of events
+:math:`\nu_{cb}(\freeset,\constrset)` as a function of unconstrained
+parameters :math:`\freeset` and constrained parameters
+:math:`\constrset`. The latter has corresponding one-dimensional
+constraint terms
+:math:`c_\singleconstr(a_\singleconstr|\,\singleconstr)` with auxiliary
+data :math:`a_\singleconstr` constraining the parameter
+:math:`\singleconstr`. The event rates :math:`\nu_{cb}` are defined as
+
+.. math::
+
+   \label{eqn:sample_rates}
+    \nu_{cb}\left(\fullset\right) = \sum_{s\in\mathrm{\,samples}} \nu_{scb}\left(\freeset,\constrset\right) = \sum_{s\in\mathrm{\,samples}}\underbrace{\left(\prod_{\kappa\in\,\bm{\kappa}} \kappa_{scb}\left(\freeset,\constrset\right)\right)}_{\text{multiplicative modifiers}}\, \Bigg(\nu_{scb}^0\left(\freeset, \constrset\right) + \underbrace{\sum_{\Delta\in\bm{\Delta}} \Delta_{scb}\left(\freeset,\constrset\right)}_{\text{additive modifiers}}\Bigg)\,.
+
+The total rates are the sum over sample rates :math:`\nu_{csb}`, each
+determined from a *nominal rate* :math:`\nu_{scb}^0` and a set of multiplicative and
+additive denoted *rate modifiers* :math:`\bm{\kappa}(\fullset)` and
+:math:`\bm{\Delta}(\fullset)`. These modifiers are functions of (usually
+a single) model parameters. Starting from constant nominal rates, one
+can derive the per-bin event rate modification by iterating over all
+sample rate modifications as shown in .
+
+As summarised in :ref:`tab:modifiers_and_constraints`, rate modifications
+are defined in :math:`\HiFa{}` for bin :math:`b`, sample :math:`s`, channel
+:math:`c`.  Each modifier is represented by a parameter :math:`\phi \in
+\{\gamma, \alpha, \lambda, \mu\}`.  By convention bin-wise parameters are
+denoted with :math:`\gamma` and interpolation parameters with :math:`\alpha`.
+The luminosity :math:`\lambda` and scale factors :math:`\mu` affect all bins
+equally.  For constrained modifiers, the implied constraint term is given as
+well as the necessary input data required to construct it.  :math:`\sigma_b`
+corresponds to the relative uncertainty of the event rate, whereas
+:math:`\delta_b` is the event rate uncertainty of the sample relative to the
+total event rate :math:`\nu_b = \sum_s = \nu^0_{sb}`.
+
+Modifiers implementing uncertainties are paired with
+a corresponding default constraint term on the parameter limiting the
+rate modification. The available modifiers may affect only the total
+number of exected events of a sample within a given channel, i.e. only
```

Comment:
`exected` should be `expected`

Reply:
Resolved in https://github.com/diana-hep/pyhf/pull/480/commits/42c558a711fa1031ea8a3d13a7297375f4b72505